gravitee-am-identityprovider/gravitee-am-identityprovider-http/src/main/resources/schemas/schema-form.json

```diff
@@ -102,6 +102,104 @@
               "exception"
             ]
           }
+        },
+        "paths": {
+          "type": "object",
+          "id": "urn:jsonschema:io:gravitee:am:identityprovider:http:HttpAuthResourcePathsConfiguration",
+          "title": "Paths",
```

Comment:
I don't understand the meaning of "Paths" in this case.
Maybe we could have "loadPreAuthUserResource" at the same level as "authenticationResource" ?

Also I'm wondering if a checkbox to enable or disable the "loadPreAuthUserResource" make sens?

Reply:
It's mainly to have clarity from the UI.

Also I'm wondering if a checkbox to enable or disable the "loadPreAuthUserResource" make sens?

No because it's not like the UserResource where something is loaded into the gateway. This one is dynamically enable or disable with the presence of the value.